pkg/api/v1alpha1/edgeconnect/edgeconnect_types.go

```diff
@@ -28,6 +28,10 @@ type EdgeConnectSpec struct { //nolint:revive
 	// Node selector to control the selection of nodes for the EdgeConnect pods
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
+	// Set custom proxy settings.
```

Comment:
```suggestion
	// General configurations for proxy settings.
```

Just because its a "bigger" section